Feliz.UseMediaQuery/UseMediaQuery.fs

```diff
@@ -0,0 +1,65 @@
+﻿module Feliz.UseMediaQuery
+
+open System
+open Browser
+open Browser.Types
+open Feliz
+
+type ScreenSize =
+    | Mobile
+    | MobileLandscape
+    | Tablet
+    | Desktop
+    | WideScreen
+
+type Breakpoints = {
+    MobileLandscape: int
+    Tablet: int
+    Desktop: int
+    WideScreen: int
+}
+
+let defaultBreakpoints = {
+    MobileLandscape = 576
+    Tablet = 768
+    Desktop = 1024
+    WideScreen = 1216
+}
+
+let makeQueries breakpoints =
```

Comment:
Please add a `private` access modifier in here because this function is internal to this module

Reply:
Fixed